backend/model/pass.model.js

```diff
@@ -30,10 +30,11 @@ const passSchema = mongoose.Schema(
       type: Boolean,
       default: false,
     },
+    timestamps: {
```

Comment:
change variable name.


https://medium.com/@taohidulii/naming-your-variables-f9477ba002e9

Reply:
changed to timeAtCheckIn